cmd/mockery.go

```diff
@@ -297,7 +297,7 @@ func getLogger(levelStr string) (zerolog.Logger, error) {
 		Out:        out,
 		TimeFormat: time.RFC822,
 	}
-	if !terminal.IsTerminal(int(out.Fd())) {
+	if !terminal.IsTerminal(int(out.Fd())) || os.Getenv("TERM") == "dumb" {
```

Comment:
I'd prefer this to be pulled in as a config value that can be bound to an env var like `MOCKERY_DISABLE_COLORING`.